src/frontend/shortlist/src/components/layout/signup_login/Login.vue

```diff
@@ -0,0 +1,126 @@
+<script>
+import NavbarLogin from "./NavbarLogin.vue";
+import Logo from "./Logo.vue";
+
```

Comment:
This is missing the pinia state manager that Pooja and Adelaide added... this whole file is quite a bit different?

https://github.com/gcivil-nyu-org/team-2-inperson/blob/develop/src/frontend/shortlist/src/components/layout/signup_login/Login.vue

Reply:
> This is missing the pinia state manager that Pooja and Adelaide added... this whole file is quite a bit different?
> 
> https://github.com/gcivil-nyu-org/team-2-inperson/blob/develop/src/frontend/shortlist/src/components/layout/signup_login/Login.vue

Ye told me pinia is a temporary solution and therefore we don’t have to worry about it.